tf_files/aws/modules/s3-logs/cloud.tf

```diff
@@ -33,6 +35,13 @@ resource "aws_s3_bucket" "log_bucket" {
   }
 }
 
+#### Added by fauzi@uchicago.edu
+
+data "aws_caller_identity" "current" {}
```

Comment:
Where is this used?

Reply:
I removed the association with the "data" resource in a previous commit and forgot to remove the invocation.

Removed now